### ISSUE TITLE
Add some notes about using the CAX data and things to know

### ISSUE THREAD

**Issue by eeholmes** [nwfsc-math-bio/rCAX#19]
I started a template for you here.
https://github.com/nwfsc-math-bio/rCAX/blob/main/inst/cax.md
Nothing extensive just some basic notes about what these data are and things to be aware of.

You can edit within GiHub if that is easiest. Click the little edit icon and then you can Preview (preview tab) while you work. To save, click the 'commit' button at the bottom.  Or move to Google Docs and I'll port the text back over the GitHub.

This will become a vignette here:
https://nwfsc-math-bio.github.io/rCAX/articles/index.html

**Comment by eeholmes** [comment]
@mari-williams Do you want to do this now before we share or postpone to version 2.0 after we get comments.

**Comment by mari-williams** [comment]
oh yeah, i looked at this and my mind went blank. I'll see if I can get some words out.

**Comment by mari-williams** [comment]
Ok, time for @katiebarnas to have a look.

**Comment by eeholmes** [comment]
@katiebarnas Can you look at https://github.com/nwfsc-math-bio/rCAX/blob/main/inst/cax.md and see if you want to add anything. Once that is done, we can close this issues.

**Comment by eeholmes** [comment]
@mari-williams It looks like the data quality section is not done. Do you want to change or delete? Here is what is there now. Seems like this is not really 'data quality', but I might be wrong. Perhaps we don't need a data quality section.

## How data quality is indicated

For the VR/SR datasets, CAX data is initially filtered for ESA listed population data by NMFS_POPID (`nmfs_popid`). POPID numbers are consecutive within an ESU. Note - this does not include superpops data, that has to be separately queried.

**Comment by mari-williams** [comment]
Yeah, I keep hacking away at it. I just confirmed that we did adopt the superpop IDs and have asked StreamNet to put the popID numbers into the NMFS_POPID column so that our filtering can pull them along with pop same/portions for our dataset.

**Comment by eeholmes** [comment]
@mari-williams I used your latest changes and then did a little clean-up on https://nwfsc-math-bio.github.io/rCAX/articles/cax.html

Is it good enough for now? We can add more later.

Note file to edit was moved from the md file to this Rmd file
https://github.com/nwfsc-math-bio/rCAX/blob/main/vignettes/cax.Rmd
